semgrep-core/Parsing/pfff/ruby_to_generic.ml

```diff
@@ -335,8 +335,9 @@ and atom x =
 and literal x =
   match x with
   | Bool x -> G.L (G.Bool (wrap bool x))
-  | Num x -> G.L (G.Int (wrap string x))
-  | Float x -> G.L (G.Float (wrap string x))
+  (* TODO: put real numbers here *)
```

Comment:
Is this meant to be left as a todo?

Reply:
yep, I was lazy and didn't finish using real numbers for Ruby, because I know the Ruby lexer is a mess right now.